WalletWasabi.Fluent/Controls/TagsBox.axaml

```diff
@@ -227,6 +227,7 @@
         <Setter Property="Margin" Value="0" />
         <Setter Property="Focusable" Value="True" />
         <Setter Property="KeyboardNavigation.IsTabStop" Value="True" />
+        <Setter Property="MaxLength" Value="{Binding Path=MaxTextLength, RelativeSource={RelativeSource AncestorType={x:Type c:TagsBox}}}" />
```

Comment:
might be able to use `{Binding $parent[c:TagsBox].MaxTextLength}` for slightly better syntax here.

Reply:
I tried to use that syntax but I couldn't make it work. I assume because the `TextBox`'s parent is the `AutoCompleteBox` and not the `TagsBox`.